src/components/badge/badge.spec.ts

```diff
@@ -1,35 +1,19 @@
-import { Badge } from './badge';
+import { abbreviate } from './format';
 
 describe('limel-badge', () => {
-    const badge = new Badge();
-    it('builds', () => {
-        expect(badge).toBeTruthy();
-    });
-
     it('Badge: returns 950 (3 digits)', () => {
-        const expectedCounter = (badge.label = 950);
-        const actualNumber = badge.showRoundedNumber();
-        expect(actualNumber).toBe(expectedCounter);
+        expect(abbreviate(950)).toBe(950);
```

Comment:
Oh yes, this is so much better! 🙏

Reply:
This should probably have been in a separate `refactor` commit since the changes were a lot more than just formatting, but they ended up in the same one in the end

Reply:
Yeah, I thought about that too, but in the end, neither `refactor` nor `style` triggers a release or shows up in the changelog, and we currently aren't using release branches with hotpatches, so I'm ok with leaving them together.

Reply:
I did split the switch from tslint to eslint out from the style-changes though. I used to think those belonged together, but actually, should we ever wish to cherry pick the tslint->eslint commit, we might very well not want to have the style changes too.

Reply:
(I might still end up splitting this out, we'll see 😄)